packages/config/src/handlers/certificate/certificateHandler.ts

```diff
@@ -41,12 +43,21 @@ export async function getActiveCertificatesHandler(
   request: Hapi.Request,
   h: Hapi.ResponseToolkit
 ) {
-  const activeCertificates = await Certificate.find({
-    status: Status.ACTIVE,
-    event: { $in: [Event.BIRTH, Event.DEATH] }
-  })
+  const token = request.headers.authorization.replace('Bearer ', '')
+  const tokenDecoded = verifyToken(token)
+  const { scope } = tokenDecoded
 
-  return activeCertificates
+  if (
+    scope.includes(RouteScope.CERTIFY) ||
+    scope.includes(RouteScope.VALIDATE)
+  ) {
+    const activeCertificates = await Certificate.find({
+      status: Status.ACTIVE,
+      event: { $in: [Event.BIRTH, Event.DEATH] }
+    })
+    return activeCertificates
+  }
+  return {}
```

Comment:
```suggestion
  return []
```
We should be either returning an array of certificates or an empty one

Reply:
Fixed